### PR TITLE
PV Rooftops YAML submission.

### PR DESCRIPTION
**Pull request by digitalrager**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/nrel-oedi-pv-rooftops.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
...ientation, and shading parameters Gagnon et al. 2016. Standard 2015 technical potential was then estimated for each plane using NREL's System Advisory Model.  The PVRDB is down-loadable by city and year of lidar collection. Five geospatial layers are available for each city and year: 1) the raster extent of the lidar collection, 2) buildings identified from the lidar data, 3) suitable developable planes for each building, 4) aspect values of the developable planes, and 5) the technical potential estimates of the developable planes. The first and current version of the PVRDB is derived from 2006-2014, 1-meter lidar data from the U.S. Department of Homeland Security for 128 metropolitan regions. Suitability is defined as any roof area that is less than 60 degrees in tilt; has an orientation facing east, southeast, south, southwest, or west; is in sunlight for a minimum number of hours per day (determined uniquely for each region); and has at least 10 m2 of contiguous area that meets the above listed criteria. Technical potential estiamtes were estimated using SAM, with NSRDB v1 as the underlying resource data. Technical parameters assumed a 2015 typical residential PV rooftop system.
```
Comment (jflasher) [inline datasets/nrel-oedi-pv-rooftops.yaml]: You'll need to add "" around this text.

File: datasets/nrel-oedi-pv-rooftops.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Documentation: https://github.com/openEDI/documentation/blob/master/PVROOFTOPS.md
+Contact:  https://github.com/openEDI/documentation/issues
+ManagedBy: National Renewable Energy Laboratory
+UpdateFrequency:  This dataset is composed of historical data only.  No new releases are expected. 
+Tags:
+  - solar
```
Comment (jflasher) [inline datasets/nrel-oedi-pv-rooftops.yaml]: Please add `aws-pds` as a tag.

File: datasets/nrel-oedi-pv-rooftops.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - lidar
+  - geospatial
+License:  Creative Commons Attribution 3.0 United States License
+Resources:
+  - Description:
```
Comment (jflasher) [inline datasets/nrel-oedi-pv-rooftops.yaml]: Please add Description and Bucket ARN like arn:aws:s3:::3kricegenome

File: datasets/nrel-oedi-pv-rooftops.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - lidar
+  - geospatial
+License:  Creative Commons Attribution 3.0 United States License
+Resources:
+  - Description: 
+    ARN:
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
```
Comment (jflasher) [inline datasets/nrel-oedi-pv-rooftops.yaml]: You can just remove this whole Tutorials block if nothing in here.

File: datasets/nrel-oedi-pv-rooftops.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      AuthorURL:  https://www.nrel.gov/
+    - Title:  Using GIS-based methods and lidar data to estimate rooftop solar technical potential in US cities
+      URL:  https://iopscience.iop.org/article/10.1088/1748-9326/aa7225/pdf
+      AuthorName:  Robert Margolis et al 2017 Environ. Res. Lett. 12 074013
+      AuthorURL: 
+    - Title:  Estimating rooftop solar technical potential across the US using a combination of GIS-based methods, lidar data, and statistical modeling
+      URL:  https://iopscience.iop.org/article/10.1088/1748-9326/aaa554/pdf
+      AuthorName:  Pieter Gagnon et al 2018 Environ. Res. Lett. 13 024027
+      AuthorURL:
+    - Title:  Rooftop Solar Photovoltaic Technical Potential in the United States: A Detailed Assessment
```
Comment (jflasher) [inline datasets/nrel-oedi-pv-rooftops.yaml]: Please add "" around the Title content.

**Comment by digitalrager** [comment]
I believe I resolved the concerns.  Thanks for the review.

**Comment by jflasher** [comment]
Will be merged in with https://github.com/awslabs/open-data-registry/pull/417

**Pull request closed without merging**